loader/include/Geode/ui/TextAreaV2.hpp

```diff
@@ -0,0 +1,241 @@
+        * Sets the padding between lines in the text area
+        */
+        void setLinePadding(const float padding);
+        /**
+        * Gets the padding between lines in the text area
+        */
+        float getLinePadding() const;
+        /**
+        * Sets a minimum bound for the width of the text area
+        *
+        * @note If this is set to a value smaller than 0, it will stop rescaling the container and leave it at the max width
+        */
+        void setMinWidth(const float width);
+        /**
+        * Resets the minimum width to prevent it from rescaling the container
+        */
+        void resetMinWidth();
+        /**
+        * Gets the min width of the text area
+        */
+        float getMinWidth() const;
+        /**
+        * Sets the max width of the text area to define the wrapping bounds
+        *
+        * @note If this is set to a value smaller than 0, it will stop rescaling the container and leave it as big as the text or the min width
+        */
+        void setMaxWidth(const float width);
+        /**
+        * Resets the maximum width to prevent it from rescaling the container
+        */
+        void resetMaxWidth();
+        /**
+        * Gets the max width of the text area
+        */
+        float getMaxWidth() const;
+        /**
+        * Sets a minimum bound for the height of the text area
+        *
+        * @note If this is set to a value smaller than 0, it will stop rescaling the container and leave it at the max height
+        */
+        void setMinHeight(const float height);
+        /**
+        * Resets the minimum height to prevent it from rescaling the container
+        */
+        void resetMinHeight();
+        /**
+        * Gets the min height of the text area
+        */
+        float getMinHeight() const;
+        /**
+        * Sets the max height of the text area to define the maximum number of lines through a height limit
+        *
+        * @note If this is set to a value smaller than 0, it will stop rescaling the container and leave it as big as the lines or the min height
+        */
+        void setMaxHeight(const float height);
+        /**
+        * Resets the maximum height to prevent it from rescaling the container
+        */
+        void resetMaxHeight();
+        /**
+        * Gets the max height of the text area
+        */
+        float getMaxHeight() const;
+        /**
+        * Resets the minimum bounds for the width and height
+        */
+        void resetMinBounds();
+        /**
+        * Resets the maximum bounds for the width and height
+        */
+        void resetMaxBounds();
+        /**
+        * Sets whether the text should be hyphenated
+        */
+        void setHyphenate(const bool hyphenate);
+        /**
+        * Gets whether the text should be hyphenated
+        */
+        bool getHyphenate() const;
+        /**
+        * Sets whether the text area should show an ellipsis when the text overflows
```

Comment:
All these doc comments should have the stars aligned to the first star of the opening `/**` using a single space

Reply:
huh, I did that but I guess vscode auto formatted it